### PR TITLE
[docs] update usage information

### PR DESCRIPTION
**Pull request by profnandaa**

### CODE REVIEW

File: src/ServiceMonitor/Main.cpp
Comment (sujitnayak) [inline src/ServiceMonitor/Main.cpp:54]: This comment is about environment variables being updated and that behavior being hardcoded.

Comment (profnandaa) [reply src/ServiceMonitor/Main.cpp:54]: got it, will revert that...

**Review by sujitnayak — APPROVED** [review]

**Comment by mloskot** [comment]
Folks, can this trivial update be merged?

Users currently have no way to learn about the features. Unless, they can read C++ code:

https://github.com/microsoft/IIS.ServiceMonitor/blob/1c4153056fd8ad8e61317af115153180dc10c2cc/src/ServiceMonitor/Main.cpp#L58-L65

**Comment by sujitnayak** [comment]
@profnandaa do you need anything else to go ahead with the merge?

**Comment by profnandaa** [comment]
@sujitnayak -- merge rights; I have limited access to the repo.

**Comment by sujitnayak** [comment]
@profnandaa Do you see any way to trigger the license/cla check? It looks like it's blocking the merge since it hasn't run yet.

**Comment by profnandaa** [comment]
Let me re-open.

**Comment by profnandaa** [comment]
Re-opened here - https://github.com/microsoft/IIS.ServiceMonitor/pull/91 @sujitnayak

**Pull request closed without merging**
